recipes/hcephes/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "hcephes" %}
+{% set version = "0.1.6" %}
+{% set sha256 = "fe9954c19b507a8a1dc79737b2fb524b20167397c146e481db3b1a18fcfeba04" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/limix/hcephes/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win32 or linux32 or (win64 and py27)]
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py36]
+
+requirements:
+  build:
+    - python  # [win]
+    - toolchain
+    - cmake
+    - vc 9   # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py36]
+  run:
+    - vc 9   # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py36]
+
+test:
+  commands:
+    - test -e $PREFIX/lib/libhcephes.so  # [linux]
+    - test -e $PREFIX/lib/libhcephes_static.a  # [linux]
+    - test -e $PREFIX/lib/libhcephes.dylib  # [osx]
+    - test -e $PREFIX/lib/libhcephes_static.a  # [osx]
```

Comment:
Please drop this line.

Reply:
This one? `- test -e $PREFIX/lib/libhcephes_static.a  # [osx]`  ? but that file is created by osx...